sprint0/sprint0/sprint0.csproj

```diff
@@ -40,7 +40,6 @@
     <Folder Include="Bosses\Gohma\" />
     <Folder Include="Bosses\Digdogger\" />
     <Folder Include="Content\LevelData\" />
-    <Folder Include="Interfaces\" />
```

Comment:
do we not need this anymore? oh well I'm sure vs will regenerate this. approved; everything else looks good